### DISCUSSION TITLE
Save Modes Now!

### DISCUSSION THREAD

**Discussion by thisiseiuai** [ScratchAddons/ScratchAddons#4456] [Ideas]
It's april fools day and they added "modes" in the editor, please add this feature as an addon once April Fools Day is over

**Comment by Norbiros** [comment]
Yes, also we can create easter egg addon to save this "rainbow" effect

**Comment by mxmou** [comment]
This could also include the "Mystery Mode" from 2020.

**Reply by CST1229** [reply]
And wobbly sprite dragging.

**Reply by mxmou** [reply]
Wobbly sprite dragging can be separate because it can be used at the same time as one of these modes.
